src/node_tests/ndt_client.js

```diff
@@ -276,8 +293,10 @@ function ndt_c2s_test() {
               die("NO DATA TRANSMITTED");
             }
             state = "DONE";
-            log("C2S test complete using " + url_protocol);
-            log("C2S rate: ", 8 * TRANSMITTED_BYTES / 1000 / (test_end - test_start));
+            log(DEBUG, "C2S test complete using " + url_protocol);
+            LOCAL_THROUGHPUT_VALUE = 8 * TRANSMITTED_BYTES / 1000 / (test_end - test_start);
+            log(DEBUG, "C2S rate: ", LOCAL_THROUGHPUT_VALUE);
+            log(INFO, "Measured upload rate of %d", LOCAL_THROUGHPUT_VALUE);
```

Comment:
Kbps, and javascript uses printf syntax so %d is an int (or, rather, the double that we pass in will be printed as an integer - all numbers is Javascript are doubles, which has some "fun" implications - https://twitter.com/pboothe/status/634469770507780097 ).